coalib/core/DependencyTracker.py

```diff
@@ -293,7 +293,7 @@ def check_circular_dependencies(self):
          ...
         coalib.core.CircularDependencyError.CircularDependencyError: ...
 
-        :raises CircularDependencyError:
+        :raises coalib.core.CircularDependencyError:
```

Comment:
I'm wondering that sphinx does not check that the docstrings contain meaningful settings.